Saucier720-app/src/app/recipes/FEC/recipe-card/recipe-card.component.html

```diff
@@ -32,7 +32,9 @@ <h2>Ingredients</h2>
                 <ul>
                   <ng-container *ngFor="let ingredient of currentIngredients; let i = index">
                     <ng-container *ngIf="!checkForRecipeFollows(ingredient)">
-                      <li *ngIf="!printedSubRecipeLines.includes(ingredient)">{{ ingredient }}</li>
+                      <li class="recipe-ingredient" *ngIf="!printedSubRecipeLines.includes(ingredient)">
+                        <div title="Add {{ ingredient }} to your list!" class="ingredient-wrapper"><button  class="not-in-list" [attr.id]="'row' + i" (click)="addToList(ingredient, $event)"><img class="list"src="/assets/images/addList.png"></button>{{ ingredient }}</div>
```

Comment:
Maybe change "Add {{ ingredient }} to your list!" to "Click to add {{ ingredient }} to your list"

Reply:
Good idea